prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -412,7 +412,7 @@ fi
 # ---------------------------------------------------------------------------------------------------------------------
 # Calibration workflows
 workflow_has_parameter CALIB && { source ${CALIB_WF:-$MYDIR/calib-workflow.sh}; [[ $? != 0 ]] && exit 1; }
-workflow_has_parameters CALIB CALIB_LOCAL_INTEGRATED_AGGREGATOR && { source ${CALIB_AGGREGATOR_WF:-$MYDIR/aggregator-workflow.sh}; [[ $? != 0 ]] && exit 1; }
+workflow_has_parameters CALIB CALIB_LOCAL_INTEGRATED_AGGREGATOR && { export AGGREGATOR_TASKS=ALL; source ${CALIB_AGGREGATOR_WF:-$MYDIR/aggregator-workflow.sh}; [[ $? != 0 ]] && exit 1; }
```

Comment:
see comment above, I'd do the check for missing `AGGREGATOR_TASKS` inside `aggregator-workflow.sh`